ocean_lib/ocean/ocean_assets.py

```diff
@@ -226,6 +226,29 @@ def _assert_ddo_metadata(metadata: dict):
 
         assert "name" in metadata, "Must have name in metadata."
 
+    @enforce_types
+    def create_url_algorithm(
+        self, name: str, url: str, publisher_wallet, wait_for_aqua: bool = True, algo_info: dict = None,
+    ) -> tuple:
+        """Create an asset of type "algorithm", with good defaults"""
+        if not algo_info:
+            algo_info = {
+                "algorithm": {
+                    "language": "python",
+                    "format": "docker-image",
+                    "version": "0.1",
+                    "container": {
+                        "entrypoint": "python $ALGO",
+                        "image": "oceanprotocol/algo_dockers",
```

Comment:
Most of `algo_info` structure is boilerplate. The one thing that isn't: the image name (`"oceanprotocol/algo_dockers"`). We want to make it easy for people to specify that.

Therefore I recommend: make `image` (image name) an argument, not `algo_info`.

Reply:
Folowwing my comments below we will need the image name and the checksum.